src/corehost/cli/fx_reference.cpp

```diff
@@ -66,40 +87,45 @@ void fx_reference_t::apply_settings_from(const fx_reference_t& from)
         set_fx_version(from.get_fx_version());
     }
 
-    const roll_fwd_on_no_candidate_fx_option* from_rollfwd = from.get_roll_fwd_on_no_candidate_fx();
-    if (from_rollfwd != nullptr)
+    const roll_forward_option* from_roll_forward = from.get_roll_forward();
```

Comment:
Why are we allowing `roll_forward_option == null` seems to unnecessarily complicate the code.